app/app.json

```diff
@@ -12,10 +12,16 @@
     "githubUrl": "https://github.com/hawkins/greene-county-alerts",
     "icon": "./assets/icon.png",
     "ios": {
-      "bundleIdentifier": "is.hawkins.greene-co-alerts"
+      "bundleIdentifier": "is.hawkins.greene"
     },
     "android": {
-      "package": "is.hawkins.greene-co-alerts"
+      "package": "is.hawkins.greene"
     }
+  },
+  "ios": {
+    "bundleIdentifier": "is.hawkins.greene"
+  },
+  "android": {
+    "package": "is.hawkins.greene"
```

Comment:
Lines 14-25 here are the alarming issue I'm describing in my commit history. I fucking hate to lay blame on anyone else but I'm almost certain its a bug on expo's end.

If it is, it'll be fixed so I'd like to revisit this eventually.

If it's not, documentation is definitely not clear about this behavior. So I could help out there.